components/cli/cmd/cellery/list_ingresses.go

```diff
@@ -51,7 +50,7 @@ func newListIngressesCommand(cli cli.Cli) *cobra.Command {
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunListIngresses(cli, args[0])
+			image.RunListIngresses(cli, args[0])
```

Comment:
Doesn't RunListIngresses method return an error?

Reply:
list ingresses command is not refactored yet, therefore RunListIngresses still does not return any error.